react/tsconfig.json

```diff
@@ -2,12 +2,10 @@
   "compilerOptions": {
     "alwaysStrict": true,
     "esModuleInterop": true,
-    "allowJs": true,
     "jsx": "react",
     "lib": ["es2017", "dom", "es2018.promise"],
     "module": "es6",
     "moduleResolution": "node",
-    "noEmitOnError": true,
     "noImplicitAny": true,
     "noImplicitReturns": true,
```

Comment:
Use the one in the recipe

Reply:
i am using the one in base-store-components